projects/libs/core-library-typescript/test/common-library/test-private-keys/secret-message-key

```diff
@@ -0,0 +1,51 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
Let's not include these at all and just generate the test keys in/before `test.sh`.